blivet/devices/lvm.py

```diff
@@ -1168,10 +1168,6 @@ def _init_check(self):
 
         if self.origin and not isinstance(self.origin, LVMLogicalVolumeDevice):
             raise ValueError("lvm snapshot origin must be a logical volume")
-        if self.origin and not self.origin.exists:
-            raise ValueError("lvm snapshot origin volume must already exist")
-        if self.vorigin and not self.exists:
-            raise ValueError("only existing vorigin snapshots are supported")
```

Comment:
The second ``if`` should stay as is, we don't want to support creation of snapshots with vorigins.